src/server/util/audit.js

```diff
@@ -0,0 +1,41 @@
+import { models } from '../models'
+
+export function getClientData(req) {
+  const userAgent = req.headers['user-agent'] || ''
+  const userId = req.user ? req.user.id : null
+  let ipAddress = req.ip
+
+  // if it includes a ':' and a '.', is's probably a "IPv4-mapped IPv6 addresses"
```

Comment:
since this is only about gathering data, we might be better off not doing this transformation either. The audit data is not something that is actively looked at, it's something that will get looked at when something's wrong. In that case, all available data will be interesting